### ISSUE TITLE
Disable ALT menu focusing

### ISSUE THREAD

**Issue by bigrando420** [EpicGamesExt/raddebugger#382]
I've got a bunch of AHK hotkeys that use alt, but they aren't working in the debugger because they're being consumed by the alt menu focusing behaviour. Is there any chance we could get an option to disable this?

**Comment by ryanfleury** [comment]
Yep, I'll add that soon. One thing I am curious about is: What behavior do you expect when using these Alt-based hotkeys in other Windows programs, like Notepad? My understanding is that the menu bar Alt hotkeys are not optional on Windows, and so I am not really sure what AHK is doing to ensure that the standard menu bar path is not picking up those Alts.

**Comment by bigrando420** [comment]
Yeah it's standard behaviour, so it's defs just a minor preference issue on my end. Most apps will just break my hotkeys, but occasionally I get lucky and there's options to disable the alt menu in the settings.
I really ought to just pick a different key to get around this issue now that I think about it.

**Comment by ryanfleury** [comment]
Added in 0.9.16.
